aten/src/ATen/native/tags.yaml

```diff
@@ -12,6 +12,12 @@
   desc: |
           This tag indicates if an operator's output's shape depends on input Tensor
           data.
+- tag: data_dependent_output
+  desc: |
+          Operator has a non-Tensor output whose value is dependent on the data
```

Comment:
Why the non-Tensor output constraint here?
This is excluding ops like `nonzero()` on purpose?

Reply:
Yes, nonzero is covered by `dynamic_output_shape`, which with symbolic shapes *is* traceable (but non-Tensor outputs are always not traceable, unless we make them return symbolic bools sometimes).

Reply:
Ok!